### DISCUSSION TITLE
Is this really still not for production use?

### DISCUSSION THREAD

**Discussion by zombiehoffa** [immich-app/immich#213] [General]
My wife and I are looking at this  with lots of excitement, is it still not safe to trust with our photos?

**Comment by alextran1502** [comment]
Even though my wife is using the app as a production user, there are still a lot of things that are needed to be worked out for performance and maintainability sake...etc, I would still advise you to treat the application in the beta stage. The upcoming update has a lot of changes in the architecture of the app, even though there will be no user data lost, I don't want to make promises that something might not go wrong :) Give it a little more time, it will be mature and stable.

We have more people that are helping to develop the application now, it can only get better.

**Reply by charleslcso** [reply]
I really look forward to replacing iCloud Shared Albums with Immich. Hopefully in the not-so-distant-future.

**Reply by jonnyparris** [reply]
Hi @alextran1502  - is there a place where the checklist / roadmap to stability can be tracked? Or similarly, is there a list of caveats / bring-your-own stability crutches that a reasonably competent coder can use alongside immich while it's in beta?

**Reply by alextran1502** [reply]
> Hi @alextran1502  - is there a place where the checklist / roadmap to stability can be tracked? Or similarly, is there a list of caveats / bring-your-own stability crutches that a reasonably competent coder can use alongside immich while it's in beta?

Hi @jonnyparris, you can search for the first release milestones on the issue page, basically those are the things we hope to achieve to get to a "stable" phase.

What do you mean on the latter part of the question?

**Reply by jonnyparris** [reply]
Aha perfect, thanks for the rapid reply! I think that answers my other question already which was along the lines of what/if/how I could supplement any critical missing features / UX with any DIY workarounds. Looking at the list, it seems like development is far enough along to adopt already, albeit with a backup solution in place.
Keen to help contribute too when time permits

**Reply by alextran1502** [reply]
Sounds good! Thank you! Happy holiday
